### ISSUE TITLE
Google-Anmeldung

### ISSUE THREAD

**Issue by christianimmler** [Rillke/Contact-Conversion-Wizard#8]
Die Google-Anmeldung funktioniert (zurzeit) nicht.
![ccw-fehler](https://user-images.githubusercontent.com/12542006/73062624-2339ce80-3e9d-11ea-91bf-5ea74a95d340.png)
Der Fehler tauchte schon früher auf. Ich bringe ihn hier nur mal wieder in Erinnerung ;-)

**Comment by tryallthethings** [comment]
Hallo! Danke für die Meldung. Hab gerade mal in den Code gesehen und soweit ich es verstehe, benutzt Contact-Conversion-Wizard für die Authentifizierung OAuth1. Dies wurde 2015 abgeschaltet und es wird nur noch OAuth2 unterstützt.

> OAuth 1.0 is no longer supported and was disabled on May 5, 2015. If your application uses OAuth 1.0, you must migrate to OAuth 2.0 to use this API.
[Details](https://developers.google.com/sheets/api/v3/authorize)

Wird wohl etwas mehr Aufwand das zu implementieren.

**Comment by deradam** [comment]
Ja, es funktioniert leider nicht. Man kann aber aus Google Contacts heraus Kontakte als vCard exportieren und dann im Wizard importieren. Das klappt ganz gut.
<img width="377" alt="grafik" src="https://user-images.githubusercontent.com/1195893/101239729-3981ff00-36ea-11eb-9d41-695f22d4dfbd.png">

**Comment by HGS2903** [comment]
Diese Lösung funktioniert aber nur ohne Kontaktbilder. Hängt dies evtl. mit der Einstellung der Google-Anmeldung zusammen? Passwort oder Smartphone Verifizierung?
Ich habe nun alles ausprobiert, aber nichts funktioniert. Wäre schön, wenn der Bug beseitigt würde.

**Comment by USchnorr** [comment]
Schließe mich dem mit gleichem Fehler an...
![2021_09_23_14_28_39_Contact_Conversion_Wizard_v3 5 0 2](https://user-images.githubusercontent.com/63063872/134506779-97f7357e-05bd-4366-bc13-98787edbf905.jpg)
.

**Comment by RufusJWB** [comment]
Ich habe ebenfalls diese Fehlermeldung erhalten
![image](https://user-images.githubusercontent.com/1633499/137164155-35599bb1-6665-4a8a-8397-4169f4c3a6ef.png)

**Comment by gituser2233** [comment]
+1

**Comment by nima-amin** [comment]
Der Fehler besteht leider immer noch, und mit dem Import der exportierten Dateien (auch ohne Bilder) komme ich in keinem Programm zurecht, da bei Googe alle Informationen in einer Spalte herauskommen, und so der Import irgendwie zu keinen leserlichen Ergebnissen führt. Ich werde wohl den Weg über den Ausdruck per pdf und manuelle Neueingabe aller 500 + Kontakte beim neuen email Hoster bzw, um sowas nochmals zu vermeiden in ein Standalone Adressbuch-Programm wie Efficcess gehen müssen ... mühsam, aber man hat sonst nichts zu tun.
